samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/TestContext.cs

```diff
@@ -17,7 +17,7 @@ public class TestContext : ITestContext, IDisposable
         private readonly IOktaSdkHelper _oktaHelper;
 
         private bool _disposed = false;
-        private const int MaxAttempts = 45; // with delay 1000 ms between attempts, it's 45 seconds of total waiting time 
+        private const int MaxAttempts = 61; // seconds
```

Comment:
It this keeps failing we might to read this from env var and if it's not there use our default.